sdk/identity/identity/src/credentials/authorizationCodeCredential.ts

```diff
@@ -3,8 +3,8 @@
 
 import qs from "qs";
```

Comment:
Same comment here about `qs`.

Reply:
We just have a separate issue to remove `qs` from identity.